ecs-init/config/common.go

```diff
@@ -98,16 +107,19 @@ func AgentTarball() string {
 	return CacheDirectory() + "/ecs-agent.tar"
 }
 
-// AgentRemoteTarball is the remote location of the Agent image, used for populating the cache. This is retrieved
-// by region and the agent filename is appended.
-func AgentRemoteTarball(region string) string {
-	baseURI := getBaseLocationForRegion(region)
-	return baseURI + AgentFilename
+// AgentRemoteBucket returns the bucket that stores the Agent image
+func AgentRemoteBucket() string {
```

Comment:
What's the intention of this method, the `AgentRemoteBucketName` is already a global variable?